components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -6066,6 +6066,11 @@ document.observe("xwiki:dom:updated", init);
               this._termDetails.removeClassName('focused');
               this._termDetails.addClassName('blurred');
               this._toggleSummarize();
+              if(!this._termDetails.down("dl &gt; *:not([style='display: none;']):not(.hidden)")) {
```

Comment:
I'm not a big fan of the `style=` check. Are you sure it works in all browsers, including IE?

Reply:
Not sure, actually. Replaced it with a different check.